platform/mbed_lib.json

```diff
@@ -42,6 +42,9 @@
         },
         "EFR32": {
             "stdio-baud-rate": 115200
+        },
+        "UNO_91H": {
+            "stdio-baud-rate": 115200
```

Comment:
Why default baud = 115200? Mbed OS default = 9600..

Reply:
we are aware that it is slow, thus an app can overwrite, just that it has been defined as 9600 is default. The other 2 families are here because there are hw limitations and can do only 115200.